nni/algorithms/compression/v2/pytorch/pruning/tools/task_generator.py

```diff
@@ -327,11 +327,12 @@ def init_pending_tasks(self) -> List[Task]:
 
     def generate_tasks(self, task_result: TaskResult) -> List[Task]:
         # initial/update temp config list
-        if self._temp_config_list is None:
+        if len(self._temp_config_list) == 0:
```

Comment:
Python's official style guide suggests to use `if not self._temp_config_list:`. But this only applies to Python `list`, not numpy ndarray.

Reply:
got it